packages/devtools/lib/src/main.dart

```diff
@@ -136,6 +140,44 @@ class PerfToolFramework extends Framework {
     screensReady.future.then(app.devToolsReady);
   }
 
+  void initSseConnection() {
+    // When running the debug DDC Build, the server won't be running so we
+    // can't connect to its API (for now at least, the API is optional).
+    if (isDebugBuild()) {
+      return;
+    }
+
+    try {
+      apiClient = ServerApiClient();
```

Comment:
apiClient is also a bit too generic a name. every client has an api. Give this a name that indicates it is the DevtoolsServerClient.

Reply:
"ServerClient" looks a bit confusing, so I just named it `devToolsServer`, so calls look like this:

```dart
devToolsServer.notifyConnected(serviceManager.service.connectedUri);
```

Seem reasonable?